src/components/events/FruitBranch.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Image from "next/image";
+import fruitBranch from "@/public/home/fruitBranch.svg";
+const FruitBranch = () => {
+  return (
+    <Image
+      src={fruitBranch}
+      alt="Fruit Branch"
+      className="positioning absolute right-0 -z-10"
```

Comment:
```suggestion
      className="absolute right-0 -z-10"
```
remove this, it does nothing

Reply:
also the svg is a bit large right now, let's reduce the width so it somewhat matches the figma, maybe w-1/4? see what works